cloverage/test/cloverage/args_test.clj

```diff
@@ -28,3 +29,26 @@
     (testing "collections are concatenated"
       (is (= '("src1" "src2")
              (:src-ns-path opts))))))
+
+(deftest parse-custom-report
+  (testing "Reading the custom-report from the args"
+    (let [crep (-> ["--custom-report" "cloverage.custom-reporter/reporter-fn"]
+                   (args/parse-args {:src-ns-path ["src"]})
+                   (first)
+                   :custom-report)]
+      (is (symbol? crep)
+          "parse the string symbol correctly")
+      (is (= 'cloverage.custom-reporter/reporter-fn crep)
```

Comment:
Wait, I'm confused. Where is this custom reporter defined?

Reply:
Either passed-in as a command-line argument (a string)
or else within the project settings as a symbol
`cloverage.custom-reporter/reporter-fn` is just a tester function to validate that we can load the symbol and call it

Reply:
On line 43 you actually make it call that custom reporter but I’m not seeing where it’s defined. I understand how you’d otherwise use it.

Reply:
Some comments on the other lines of code that are involved